src/unix/linux_like/emscripten/align.rs

```diff
@@ -37,10 +37,12 @@ macro_rules! expand_align {
                 size: [u8; ::__SIZEOF_PTHREAD_COND_T],
             }
 
+            // Lowered from 16 to 8 bytes in
+            // https://github.com/llvm/llvm-project/commit/d1a96e906cc03a95cfd41a1f22bdda92651250c7
             #[allow(missing_debug_implementations)]
-            #[repr(align(16))]
+            #[repr(align(8))]
```

Comment:
(Irrelevant remark.) The stack still has to be aligned to 16 bytes when you call a function even though maxalign is only 8.  Discovering this wasted a lot of my time.